src/pages/library/DataBrowserToggler.js

```diff
@@ -0,0 +1,52 @@
+import { div, h, strong } from 'react-hyperscript-helpers'
+import { Link, Switch } from 'src/components/common'
+import colors from 'src/libs/colors'
+import { isDataBrowserVisible } from 'src/libs/config'
+import * as Nav from 'src/libs/nav'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+import { catalogPreviewStore } from 'src/pages/library/dataBrowser-utils'
+
+
+export const DataBrowserPreviewToggler = ({ checked }) => {
+  const { user: { id } } = useStore(authStore)
+  catalogPreviewStore.set({ [id]: checked })
+
+  return !isDataBrowserVisible() ? div() : div({
+    style: {
+      background: colors.dark(0.1),
+      padding: '10px 15px',
+      margin: 15,
+      border: '1px solid', borderColor: colors.accent(), borderRadius: 3,
+      display: 'flex', flexDirection: 'row'
+    }
+  }, [
+    div({ style: { display: 'flex', flexDirection: 'column' } }, [
+      strong(['Toggle to preview the new Data Catalog']),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: 6 } }, [
+        h(Switch, {
```

Comment:
That's true, I think it does feel kind of unexpected... but in the browser, it's not entirely obvious that it's a new page loaded, does it? It looks like you are just toggling a setting, since the tab doesnt actively look like it's changed

Reply:
It doesn't _look_ like it, but the navigation is noticeable in other ways. Focus gets moved back to the top of the page and if you're using a screen reader, it reads the new page. I leave it up to you which way to go.

However, one thing I noticed while trying this with a screen reader is that the switch doesn't have a label. It currently reads as "off, switch" or "on, switch". There's already some descriptive text here, it just needs to be wrapped in a `label` tag with the Switch component. The usage section on https://www.npmjs.com/package/react-switch has an example.